frontend/public/components/overview/deployment-overview.tsx

```diff
@@ -9,6 +9,7 @@ import { OverviewDetailsResourcesTab } from './resource-overview-page';
 import { ResourceOverviewDetails } from './resource-overview-details';
 import PodRingSet from '@console/shared/src/components/pod/PodRingSet';
 import { OverviewItem } from '@console/shared';
+import MonitoringTab from '@console/dev-console/src/components/monitoring/overview/MonitoringTab';
```

Comment:
we can't import `dev-console` here, need to see how we can handle with extensions